### PR TITLE
[1LP][RFR]Add request.addfinalizer wherever missing in reports

### PR DESCRIPTION
**Pull request by valaparthvi**
## Purpose or Intent
- __Extending__ reports tests by adding finalizer wherever missing.

### PRT Run
{{ pytest: cfme/tests/intelligence/reports/test_crud.py cfme/tests/intelligence/reports/test_views.py -vvv }}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 069804fba76c86bbfea19d992cba30ad245662ac

The *local* fixture **`report`** is used in the following files:
- **cfme/tests/intelligence/reports/test_views.py**
    - *test_report_view*

Please, consider creating a PRT run against these tests make sure your fixture changes do not break existing usage :smiley:

**Comment by valaparthvi** [comment]
Moving it to [RFR] since PRT passed for 510 and 3 tests that failed for 511 are related to widget and dashboard and the failures are unrelated to my changes.

File: cfme/tests/intelligence/reports/test_crud.py [hunk 1]
```diff
@@ -333,6 +347,14 @@ def test_import_report_rest(appliance):
     }
     response, = appliance.rest_api.collections.reports.action.execute_action("import", data)
     assert_response(appliance)
+
+    @request.addfinalizer
+    def _finalize():
+        report = appliance.collections.reports.instantiate(
```
Comment (digitronik) [inline cfme/tests/intelligence/reports/test_crud.py]: Q: it is not possible to remove `report` with `rest` as test pointing rest?

Comment (valaparthvi) [reply cfme/tests/intelligence/reports/test_crud.py]: It's not possible, we can only `run` and `schedule` a report entity via REST and `import` a report with ReportCollection via REST..

**Pull request merged**